__tests__/plugins/network.test.ts

```diff
@@ -278,4 +278,18 @@ describe('network', () => {
       expect(timing.total).toBeLessThan(50);
     });
   });
+
+  it("doesn't capture network info from request context", async () => {
```

Comment:
I checked PW and its not exposed correctly for our use case. Have to do a PR upstream to make it work.